components/layout_2020/flexbox/layout.rs

```diff
@@ -2137,14 +2192,25 @@ impl FlexItemBox {
             Direction::Block
         };
 
+        let cross_size =
+            if content_box_size.cross.is_auto() && auto_cross_size_stretches_to_container_size {
+                let outer_cross_size = if cross_axis_is_item_block_axis {
+                    containing_block.size.block
+                } else {
+                    containing_block.size.inline
+                };
+                outer_cross_size.map(|v| v - pbm_auto_is_zero.cross)
```

Comment:
```suggestion
               if cross_axis_is_item_block_axis {
                    containing_block.size.block
                } else {
                    containing_block.size.inline
                }.map(|v| v - pbm_auto_is_zero.cross)
```

Reply:
It would be nice to add some sort of mapping in another PR that would make this easier. Imagine something like this:

```rust
    containing_block.size
        .get_axis(self.axes.cross)
        .map(|v| v - pbm_auto_is_zero.cross)
```

where `self.axes` is `FlowRelativeVec2<Direction>`.